lib/routes.js

```diff
@@ -72,7 +82,10 @@ export default function routes(req, res, logger) {
     log.end().addDefaultFields(clientInfo);
 
     if (req.url === '/_/healthcheck') {
-        return setHealthCheckResponse(req, res, log);
+        if (!checkIP(clientInfo.clientIP)) {
+            return writeResponse(403, res, log);
```

Comment:
You should use arsenal error as we do for other routes. `AccessDenied` would be the best fit here.
```js
import { errors } from 'arsenal';
....
const error = errors.AccessDenied;